app/policies/casa_case_policy.rb

```diff
@@ -89,35 +88,44 @@ def permitted_attributes
   end
 
   def same_org_supervisor_admin_or_assigned?
-    is_in_same_org? && (
-    admin_or_supervisor? || is_volunteer_actively_assigned_to_case?
-  )
+    admin_or_supervisor_same_org? || is_volunteer_actively_assigned_to_case?
   end
 
   def same_org_supervisor_admin?
-    is_in_same_org? && admin_or_supervisor?
+    admin_or_supervisor_same_org?
   end
 
   def index?
-    admin_or_supervisor_or_volunteer?
+    admin_or_supervisor_or_volunteer_same_org?
   end
 
   alias_method :show?, :same_org_supervisor_admin_or_assigned?
-  alias_method :save_emancipation?, :index?
+  alias_method :save_emancipation?, :index? # Should this be the same as edit?
   alias_method :edit?, :same_org_supervisor_admin_or_assigned?
   alias_method :update?, :same_org_supervisor_admin_or_assigned?
   alias_method :new?, :same_org_supervisor_admin?
   alias_method :create?, :same_org_supervisor_admin?
   alias_method :destroy?, :same_org_supervisor_admin?
 
+  # TODO:
+  # The dashboard policy only shows the "New Case" button to admins, but here
+  # we allow supervisors to also access new. View spec specifically tests the
+  # casa_cases/new.html.erb view against supervisors.
+  #
+  # Should supervisors be able to create cases?
+
   private
 
+  # This method is no longer used, candidate for removal.
```

Comment:
Instead of commenting, we should remove it and see if the tests / QA pass.

Reply:
Thanks for the note! This method is actually no longer used as a result of the changes we made, and the suite passes so we can definitely remove the method.